cli/command/image/build_buildkit.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/moby/buildkit/util/progress/progressui"
 	"github.com/pkg/errors"
 	fsutiltypes "github.com/tonistiigi/fsutil/types"
+	"github.com/tonistiigi/go-rosetta"
```

Comment:
can you add a license to github.com/tonistiigi/go-rosetta ?

Reply:
Or would this be code we want to roll up into (e.g.) https://github.com/moby/moby/tree/cadd72d37d57e2dc686502d9f9ab0fc73d3b6af8/pkg/parsers